content/docs/v2/_dev/operations/monitoring.md

```diff
@@ -41,22 +57,278 @@ otelcol_receiver_accepted_spans{
+      service.name: jaeger
+      service.version: "2.0.0"
+```
+
+#### Component-Specific Debug Logging
+Configure debug logging with processor settings:
+
+```yaml
+service:
+  telemetry:
+    logs:
+      level: debug
+      development: true
+      encoding: console
+      disable_caller: false
+      disable_stacktrace: false
+
+processors:
+  batch:
+    timeout: 1s
+    send_batch_size: 1024
+    send_batch_max_size: 2048
+  memory_limiter:
+    limit_mib: 1000
+    spike_limit_mib: 500
+    check_interval: 5s
+```
+
+#### Environment Variable Override
+Alternative log level configuration for deployments:
+
+```yaml
+# Docker/Kubernetes deployment
+apiVersion: apps/v1
+kind: Deployment
+spec:
+  template:
+    spec:
+      containers:
+      - name: jaeger-collector
+        env:
+        - name: OTEL_LOG_LEVEL
+          value: "debug"
+```
+
+#### Storage Backend Specific Logging
+
+**Elasticsearch:**
+```yaml
+exporters:
+  elasticsearch:
+    endpoints: ["http://elasticsearch:9200"]
+    traces_index: jaeger-traces-{2006-01-02}
+    logs_index: jaeger-logs-{2006-01-02}
+    sending_queue:
+      enabled: true
+      num_consumers: 10
+      queue_size: 1000
+```
+
+**OpenSearch:**
+```yaml
+exporters:
+  opensearch:
+    http:
+      endpoints: ["https://opensearch:9200"]
+      tls:
+        insecure_skip_verify: true
+    dataset: traces
+```
 
 ## Traces
 
 Jaeger has the ability to trace some of its own components, namely the requests to the Query service. For example, if you start `all-in-one` as described in [Getting Started](../../getting-started/), and refresh the UI screen a few times, you will see `jaeger` populated in the Services dropdown.
 
 Self-tracing can be disabled by setting `OTEL_TRACES_SAMPLER=always_off` environment variable.
 
-## Monitoring Best Practices
+### Internal Tracing Configuration
 
-When running Jaeger in production, comprehensive monitoring is essential to ensure optimal performance and reliability. This section covers key metrics and monitoring strategies.
+#### Enable Self-Tracing
+Configure Jaeger to trace its own operations:
 
-### Process-Level Metrics
+```yaml
+service:
+  telemetry:
+    traces:
+      level: detailed
+      processors:
+        - batch
+        - memory_limiter
+
+processors:
+  memory_limiter:
+    limit_mib: 512
+  batch:
+    timeout: 1s
+    send_batch_size: 1024
+
+exporters:
+  otlp/internal:
+    endpoint: http://jaeger-internal:4317
+    tls:
+      insecure: true
+```
+
+#### Trace Sampling for Internal Operations
+Control sampling for Jaeger's internal traces:
+
+```yaml
+processors:
+  probabilistic_sampler:
+    sampling_percentage: 1
+```
+
+## Health Checks
+
+Jaeger v2 supports health monitoring through OpenTelemetry Collector health check extensions.
+
+### Health Check Extension (Standard)
+Basic health monitoring:
+
+```yaml
+extensions:
+  health_check:
+    endpoint: 0.0.0.0:13133
+    path: "/health"
+
+service:
+  extensions: [health_check]
```

Comment:
are you just making up these examples? Please prove that this is a working config.